geopmdpy/geopmdpy/stats.py

```diff
@@ -132,5 +200,9 @@ def reset(self):
         global _dl
         self._check_ptr('reset')
         err = _dl.geopm_stats_collector_reset(self._collector_ptr)
+        self._is_reset = True
         if err < 0:
             raise RuntimeError('geopm_stats_collector_reset() failed: {}'.format(error.message(err)))
+
+    def is_reset(self):
```

Comment:
Consider adding "is_reset()" method to C++ Collector, not just the python interface